conf/modules/relativelocalizationfilter.xml

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="relativelocalizationfilter" dir="relativelocalizationfilter">
+	<doc>
+		<description>
+			Relative Localization Filter based on Bluetooth Communication between drones
```

Comment:
Does this filter only work with bluetooth or can all RSSI measurements be used? Can you add a more comprehensive description of what this module is doing?